cachelib/allocator/tests/BaseAllocatorTest.h

```diff
@@ -4841,7 +4841,7 @@ class BaseAllocatorTest : public AllocatorTest<AllocatorT> {
 
       std::memcpy(newItem.getMemory(), oldItem.getMemory(), oldItem.getSize());
       ++numMoves;
-    }, {}, 1000000 /* lots of moving tries */);
```

Comment:
why change this?

Reply:
This change just returns the test to the initial state (like in main branch).
@byrnedj increased the number of tries, but after various fixes looks like it is not required anymore.